### ISSUE TITLE
Steam App Store Listing

### ISSUE THREAD

**Issue by XenHat** [Open-Shell/Open-Shell-Menu#587]
_[Asked in /discussions/582 by @H3rmit]_
 **Could you please put this on steam without any DRMs?**

I haven't considered this until now, but this is an interesting idea.
We would need to investigate if the modifications we do to the windows interface are allowed by the Steam rules.

Also, until we fully separate the explorer and IE components to offer them as **free** "DLC", I say we only publish the menu functionality on Steam, for seamless install without a dialog to click through.

**Comment by coddec** [comment]
Interesting idea for sure

**Comment by namazso** [comment]
non-game software is [currently not accepted on steam](https://help.steampowered.com/en/wizard/HelpWithPublishing?issueid=925):

![image](https://user-images.githubusercontent.com/8676443/103431867-5e796d00-4bd7-11eb-8f35-5c5c6c4927b3.png)

**Comment by XenHat** [comment]
Perhaps when we gathered enough popularity... I know a few exceptions.
https://store.steampowered.com/app/620050/Start10/
https://store.steampowered.com/sub/323175/

**Comment by namazso** [comment]
@XenHat  both of those were listed before Steam Direct, in the time of Greenlight. Obviously, they haven't removed apps already on store on transition.
